1_fetch/src/download_from_s3.R

```diff
@@ -0,0 +1,10 @@
+download_s3_file <- function(target_name, s3_bucket, s3_file_loc, s3_filename) {
+  
+  # Setup credentials
+  aws.signature::use_credentials(profile='default', file=aws.signature::default_credentials_file())
```

Comment:
I think you can probably just run this once, and use `save_object` directly in the remake file.  This isn't really hurting anything though.